src/ui/components/views/Home.web.js

```diff
@@ -3,15 +3,20 @@
 import React, { Component, PropTypes } from 'react';
 import Radium from 'radium';
 import Colors from '../../Colors';
+import fs from 'fs';
+
+
+const promo = fs.readFileSync(__dirname + '/../../../../templates/promo.html');
```

Comment:
What solution do you suggest?

- `require(raw!` won't work on the server
- putting all the static HTML as React Nodes mangles the inline SVG (see the other comment)
- there are 8 SVGs, it will load considerably slower without inlining

The current requirement (actually last Friday’s) is to have the server rendering working properly; at this point this file isn't bundled with webpack so this is a future, not present, problem.

If and when we build a client-side web app, we can address it.